test/integration/rules/aria-dialog-name/aria-dialog-name.html

```diff
@@ -0,0 +1,33 @@
+<!-- PASS -->
+<dialog id="pass1" aria-label="Cookies" />
+<dialog id="pass2" title="Cookies" />
+<dialog id="pass3" aria-labelledby="dialog-heading">
+	<h2>Cookies</h2>
+</dialog>
+<div id="pass4" role="dialog" aria-label="Cookies" />
+<div id="pass5" role="dialog" title="Cookies" />
+<div id="pass6" role="dialog" aria-labelledby="dialog-div-heading">
```

Comment:
This and the other `aria-labelledby` elements aren't pointing to valid IDs in the document. To fix, we could move the child `<h2>`s into the root and give them an ID to match the `aria-labelledby` value:

```html
<div id="pass6" role="dialog" aria-labelledby="dialog-div-heading"></div>
<h2 id="dialog-div-heading">Cookies</h2>
```

Reply:
Yep, found it already and pushed!